integration-tests/rh-push-to-registry-redhat-io/resources/tenant/sa-rolebinding.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: release-pipeline-resource-role-binding-for-${component_name}
+  namespace: ${tenant_namespace}
+  labels:
+    originating-tool: "${originating_tool}"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: release-pipeline-resource-role
+subjects:
+  - kind: ServiceAccount
+    name: ${managed_sa_name}
+    namespace: ${managed_namespace}
+  - kind: ServiceAccount
+    name: appstudio-pipeline
```

Comment:
I thought this SA was deprecated (I'm now noticing most if not all of the new tests use it too)
https://issues.redhat.com/browse/KONFLUX-5207

Reply:
I can remove for all in a subsequent PR once this merges. sound ok?

Reply:
great, thanks!

Reply:
fixed by #1292